packages/connect/src/index.ts

```diff
@@ -1,23 +1,21 @@
 export { HttpClient } from './http_client';
 export { ordersChannelFactory } from './orders_channel_factory';
-export {
-    Client,
+export { Client, OrdersChannel, OrdersChannelHandler } from './types';
+import {
```

Comment:
So if you want to use any of these types you need to add sra-types now too? Same question as above.

Reply:
Ok I see you answered this in the description. I guess the only question is why new package instead of just `types`

Reply:
And should connect re-export these types for convenience?